aviary/examples/external_subsystems/engine_NPSS/NPSS_Model/Maps_and_Elements/3Dmaps.int

```diff
@@ -0,0 +1,473 @@
+#ifndef __TURBINE_PR_MAP3D__
+#define __TURBINE_PR_MAP3D__
+
+#include <InterpIncludes.ncp>
+
+// Variables that exist in the parent Turbine element
+// and are used in this subelement.
+extern real Np;
+extern real Wp;
+extern real WpCalc;
+extern real NpDes;
+extern real WpDes;
+extern real effDes;
+extern real PRdes;
+
+// Variables that exist in the parent Turbine element
+// and are set by this subelement.
+extern real WpBase;
+extern real effBase;
+extern real PRbase; // set by ind_PRbase
+
+class TurbinePRmap3D extends Subelement {
+
+//------------------------------------------------------------
+//     ******* DOCUMENTATION *******
+//------------------------------------------------------------
+
+  title = "";
+
+  description = isA() + " manages the calculation of turbine performance
+  for an efficiency-based map.  It performs calls to maps that return
+  efficiency and corrected weight flow as a function of correct speed and
+  pressure ratio.";
+
+
+  usageNotes = isA() + 
+" 
+
+- NOTE TO USERS: The turbine subelement architecture has changed as of NPSS
+version 2.4.  What used to be the TurbineNeppMap subelement is now the
+TurbinePRmap subelement, and what used to be the TurbineEfficiencyMap
+subelement has been absorbed by the new TurbinePRmap subelement.  This
+single subelement was created to simplify the operations performed in the
+turbine S_map socket.  Only a few attributes � such as S_hum socket, effDes
```

Comment:
Maybe it's just me, but here and elsewhere in this file there appears to be some missing/broken symbol?
![image](https://github.com/OpenMDAO/Aviary/assets/51181861/a8179436-48f7-41e2-bd2d-bc2adb00864b)

Reply:
I feel like I commented on this already, but do not see it. So forgive me if this is a repreat post.    I ran through the files with bad symobls, updated them, and fixed a file that was recreating them. the change is uploaded.